onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -5752,6 +5752,7 @@ def aten_std_mean(self: TensorType, unbiased: bool = True) -> tuple[TensorType,
     raise NotImplementedError()
 
 
+@torch_op("aten::stft", trace_only=True)
```

Comment:
I wonder if we should add is_complex=True to the decorator

Reply:
We can either do that, or keep `is_complex` attribute in TorchScriptTensor.

Reply:
I think we need both? Since this is telling the system the op is handling complex numbers. Edit: I commented in the wrong place. This is more applicable to `abs_complex` whose input is complex.

Reply:
Sure, we can have another registration for complex function. I was thinking of finding the function with tail of _complex, which is not so rigorous.

Reply:
Agree. I don’t think we should make any assumptions on the names, which don’t carry any special meanings